apps/frontend/src/views/Results.vue

```diff
@@ -456,6 +473,16 @@ export default class Results extends Vue {
       this.eval_info = index;
     }
   }
+
+  // Expand all visible controls
+  expand_all() {
+    (this.$refs.controlTable as Vue & { expandAllEvaluations: () => void }).expandAllEvaluations();
+  }
```

Comment:
Moving the 'Expand All' toggle to ControlRow should should remove the need for this.